spec/attribute.dd

```diff
@@ -905,7 +909,7 @@ void main() @nogc
 )
 
 
-$(H2 $(LNAME2 class-attributes, Class Attributes))
+$(H2 $(LNAME2 oop-attributes, OOP Attributes))
```

Comment:
I don't like this rename, it breaks existing links to this part of the spec, and I think it's clear that interfaces fall under the category of classes.

Reply:
I can keep the anchor (it was only added just before I submitted this). The reason I changed heading to OOP is because synchronized is only a class or method attribute, so you might expect to find it under class attributes rather than shared storage attributes.

Reply:
> it was only added just before I submitted this

Oh okay, nevermind then.